test/integration/good/code-gen/dune

```diff
@@ -18,4 +18,15 @@
 
 (alias
  (name runtest)
- (action (diff mir.expected mir.output)))
\ No newline at end of file
+ (action (diff mir.expected mir.output)))
+
+ (rule
+ (targets cl.output)
+ (deps (package stanc) (:stanfiles  glm.stan))
+ (action
+  (with-stdout-to %{targets}
+   (run %{bin:run_bin_on_args} "%{bin:stanc} --print-cpp --use-opencl" %{stanfiles}))))
```

Comment:
@seantalts Can I add this rule? Or is there some other way of doing this?

Reply:
Looks good to me :)

Reply:
To add a bit more context. I made a glm.stan file that has all the possible signatures and different variations of when the first and second argument are data or parameters. This should be tested I guess (with --use-opencl).